src/libraries/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -259,6 +259,11 @@ private void GenerateFile(List<string> typeNames, string assemblyName, bool prox
                 {
                     if (type != null)
                     {
+                        if (verbose)
+                        {
+                            Console.WriteLine($"Importing {type.Name} ({i + 1}/{types.Length})");
```

Comment:
Please add to strings.resx and use SR.Format so that this message can be localized for other languages.

Reply:
Done, thanks.